mrp_project_wo_timesheet/models/mrp_production.py

```diff
@@ -0,0 +1,21 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import fields, models
+
+
+class ManufactureOrder(models.Model):
+    _inherit = 'mrp.production'
+
+    timesheet_ids = fields.One2many('account.analytic.line', 'mrp_production_id')
+    timesheet_count = fields.Integer(compute='_compute_timesheet_count')
```

Comment:
These field names are too generic. Another addon could add them later and cause conflict. As these are "timesheets generated from workcenter productivities", a possible renaming could be this one, what do you think?
```suggestion
    wp_timesheet_ids = fields.One2many('account.analytic.line', 'mrp_production_id')
    wp_timesheet_count = fields.Integer(compute='_compute_wp_timesheet_count')
```